docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,610 +1,131 @@
-7. In the `Cryptography` tab change `Provider Category` to `Key Storage Provider`, then `Algorithm name` to `ECDH_P384`. Also, change `Request hash` to `SHA384`.
-8. Next, in the `Extensions` tab select `Application Polices` and click the `Edit` button.
-9. Remove all entries from the list.
-10. Go to the `Security` tab, select `Domain Controllers` and give the group `Read` and `Enroll` permissions.
-11. Finally, create a template by clicking `OK`.
-12. Go back to the Certificate Authority window and right-click on `Certificate Templates`. Then:
-
-```text
-New > Certificate Template to Issue
-```
-Select `RemoteDesktopAccess` and click `OK`.
-
-### Update GPO to use a new certificate template
-
-In the group policy editor for `Teleport Access Policy`, select:
-
-```
-Computer Configuration > Policies > Administrative Templates > Windows Components > Remote Desktop Services > Remote Desktop Session Host > Security
-```
-
-Right-click on `Server authentication certificate template`, `Edit`, then select `Enabled` and fill `Certificate Template Name` with `RemoteDesktopAccess`.
-
-<Figure align="left" bordered caption="RDP Certificate Template">
-  ![RDP Certificate Template](../../img/desktop-access/rdp-certificate-template.png)
-</Figure>
-
-
-### Configure server certificate auto-enrollment
-
-In the group policy editor for `Teleport Access Policy`, select:
-
-```
-Computer Configuration > Policies > Windows Settings > Public Key Policies
-```
+</Details>
 
-Double-click on `Certificate Services Client - Auto-Enrollment`, then select `Enabled` in the `Configuration Model`.
+This guide will help you configure Teleport and Active Directory in order to provide secure, passwordless access to Windows desktops.
 
+## Prerequisites
 
-### Ensure your GPO is updated
+<ScopedBlock scope={["oss", "enterprise"]}>
 
-If you have not done so already, ensure your GPO is updated by opening a PowerShell prompt and running:
+- One or more hosts to run the Teleport Auth and Proxy services on.
 
-```powershell
-gpupdate.exe /force
-```
+</ScopedBlock>
+- A server or virtual machine running a Windows Server operating system.
 
-## Step 5/7. Export your LDAP CA certificate
+- A Linux host where you will run the Teleport Desktop Service.
+  <ScopedBlock scope={["oss", "enterprise"]}> This guide assumes that you will
+  run Teleport's Windows Desktop Service on a dedicated host. To install Desktop
+  Access into an existing Teleport instance running other services, see the
+  [Manual Setup](manual-setup.mdx) guide for Desktop Access.</ScopedBlock>
 
-Teleport connects to your Domain Controller via LDAPS. This means that you must
-let Teleport know that the certificate sent by your Domain Controller during the
-initial SSL connection is trusted. If your Domain Controller's certificate is
-trusted by the system repository on the system running Teleport, you can skip
-this step.
+  See [Installation](../installation.mdx) for details.
```

Comment:
Upon reflection, I think we can remove this line since we go through installation using partials.
```suggestion
```